contrib/completion/bash/docker

```diff
@@ -1890,15 +1890,29 @@ _docker_save() {
 }
 
 _docker_search() {
+	local key=$(__docker_map_key_of_current_option '--filter|-f')
+	case "$key" in
+		is-automated)
+			COMPREPLY=( $( compgen -W "true false" -- "${cur##*=}" ) )
+			return
+			;;
+		is-official)
+			COMPREPLY=( $( compgen -W "true false" -- "${cur##*=}" ) )
+			return
+			;;
```

Comment:
Should we change it to just "stars"? Thinking of it, "has-stars" sounds like it's a Boolean indeed